packages-ts/gauntlet-terra/src/commands/internal/terra.ts

```diff
@@ -130,6 +134,38 @@ export default abstract class TerraCommand extends WriteCommand<TransactionRespo
     return this.wrapResponse(res)
   }
 
+  async migrateContract(contract: AccAddress, newCodeId: number, migrateMsg: any) {
```

Comment:
Would like for us to move this implementation to the MigrateContract class, similar to how UpdateContractAdmin (and the rest of CMDs) are implemented.

Reply:
This was not moved to the `MigrateContract`, why not?

Reply:
Aha, this is not used anymore?